designer/server/src/lib/forms.js

```diff
@@ -125,6 +125,28 @@ export async function createDraft(id, token) {
   return response
 }
 
+/**
+ * Updates a metadata object.
+ * @param {string} id
+ * @param {Partial<FormMetadataInput>} metadata
+ * @param {string} token - auth token
+ * @returns
+ */
+export async function updateMetadata(id, metadata, token) {
+  const patchJsonByType = /** @type {typeof patchJson<FormMetadataInput>} */ (
```

Comment:
Ahh! Mind checking this? I think we return `{ id, slug, status }`?

We definitely set the wrong `body` type already in places and might be spreading it further here

Reply:
Good spot, thanks. I wasn't even using the return value which is why I didn't spot it before.

I've added a type to the model and set it here. Fixed.

Reply:
Great, thanks for this

Can we change the name `FormUpdateResponse` though?

We've got the same response via **POST** when creating forms too

Reply:
Done